driver/pom.xml

```diff
@@ -68,6 +68,17 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I had to change the version, cause I was getting some methodNotFoundExceptions since mockito-all embeds hamcrest